example/docker/amd64-musl-gcc/dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:alpine AS builder
+
+RUN apk update && apk add --no-cache git
+
+RUN git clone https://github.com/enfein/mieru.git /build
+
+WORKDIR /build
+
+RUN go mod tidy
+
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOCC=musl-gcc go build -ldflags="-s -w" -o mita cmd/mita/mita.go
```

Comment:
The `GOCC=musl-gcc` have no effect and should be removed, because we have `CGO_ENABLED=0`, which means there is no C compiler needed and C library linked. Please help to check if the program still runs after it is removed.